src/components/TextAreaInput/TextAreaInput.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { useFormContext } from "react-hook-form";
+import { FormValues } from "../Form/Form";
+
+interface TextAreaInputProps {
+  name: keyof FormValues; // Restrict name to keys of FormValues
+  placeholder?: string;
+  error?: { message: string };
+}
+
+const TextAreaInput: React.FC<TextAreaInputProps> = ({ name, placeholder }) => {
+  const {
+    register,
+    formState: { errors },
+  } = useFormContext<FormValues>();
+
+  const errorMessage = errors[name]?.message as string | undefined;
+
+  return (
+    <div className=" w-full flex flex-col items-center">
+      <textarea
+        className="w-full h-[167px] rounded-[12px] p-3.5 bg-offWhite mb-2.5 box-border 
```

Comment:
I left a similar comment on Joshua's form input element regarding the default blue outline that appears when the input is active. We weren't given specific styling for that on the figma and we would want to be consistent across input elements, but I thought we could try to override the default blue with the lightGrey that is in the tailwind. This is optional, and considering the short timeline seems fine to just stick to the default blue.
<img width="1080" alt="Screenshot 2024-09-15 at 12 24 14 PM" src="https://github.com/user-attachments/assets/7a543bb6-5ca2-4f46-827d-b66697b93a8d">